mako.systemd.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Lightweight Wayland notification daemon
+Documentation=man:mako(1)
+PartOf=graphical-session.target
+
+[Service]
+Type=dbus
+BusName=org.freedesktop.Notifications
+ExecCondition=/bin/sh -c '[ -z $WAYLAND_DISPLAY ] && exit 1 || exit 0'
```

Comment:
Can simplify to:

```
[ -n "$WAYLAND_DISPLAY" ]
```

Reply:
Hint: You can just add an inline suggestion in these review-comments for tiny changes like that.